lib/app/modules/order/components/track/components/cancel/class/cancel_store.dart

```diff
@@ -1,7 +1,7 @@
 import 'package:mobx/mobx.dart';
 part 'cancel_store.g.dart';
 
-class CancelStore = _CancelStoreBase with _$Cancelstore;
+class CancelStore = _CancelStoreBase with _$CancelStore;
```

Comment:
- Quando for mudar um codigo de outro grupo da um toque, so pra nós ficaramos sabendo

Reply:
Essa mudança se fez necessária por um erro de escrita. _$Cancelstore com "s" minúsculo era reportada como inexistente, mas _$CancelStore não. Imaginei que o erro era decorrente do merge e agi alterando.

- Vou me certificar de passar adiante a palavra caso algo assim seja necessário novamente.